pushover.sh

```diff
@@ -31,6 +32,7 @@ usage() {
     echo " -u <url>"
     echo " -U <USER> (required if not in config file)"
     echo " -a <url_title>"
+	echo " -P <http://proxy:port/>"
```

Comment:
Hey, I've tried that with the default syntax of curl but it didn't worked, because the which could not detect the curl path. And my tab references to 4 spaces =) But what I want to do is also that that the -P option accepts socks proxys too, will do that as soon as I have time for it =)